android/app/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.thealphamerc.flutter_twitter_clone">
+    package="com.flutter_twitter_clone">
```

Comment:
@aa-rodriguezv package should not be changed, so please revert this to the old one.

Reply:
Fixed back to original